libnetwork/internal/rootlessnetns/netns_linux.go

```diff
@@ -133,7 +140,7 @@ func (n *Netns) cleanup() error {
 	if err := netns.UnmountNS(nsPath); err != nil {
 		multiErr = multierror.Append(multiErr, err)
 	}
-	if err := n.cleanupSlirp4netns(); err != nil {
+	if err := n.cleanupRootlessNetns(); err != nil {
 		multiErr = multierror.Append(multiErr, wrapError("kill slirp4netns", err))
```

Comment:
the wrapError message needs to change as well, maybe to `kill networking process`?